src/pages/Countries/components/CountriesDisplay/components/CountryFlag/CountryFlag.css

```diff
@@ -2,32 +2,53 @@
   --background-color-dark: rgb(43, 57, 69);
```

Comment:
te zmienne w sumie warto wrzucić w jakiś globalny css (np. theme.css) - można jeszcze się bawić w podział tego theme na kolory i inne osobno (każda właściwość osobny plik), ale dla uproszczenia niech to będzie jeden plik